Pages/Text-utilities/textUtilities.css

```diff
@@ -0,0 +1,49 @@
+.heading1 {
+    font-size: 3rem;
+    line-height: 4rem;
+    font-weight: 900;
+}
+
+.heading2 {
+    font-size: 2rem;
+    line-height: 2.5rem;
+    font-weight: 900;
+}
+
+.heading3 {
+    font-size: 1.5rem;
+    line-height: 2rem;
+    font-weight: 800;
+}
+
+.heading4 {
+    font-size: 1rem;
+    line-height: 1.5rem;
+    font-weight: 700;
+}
+
+.heading5 {
+    font-size: 0.8rem;
+    line-height: 1rem;
+    font-weight: 600;
+}
+
+.heading6 {
+    font-size: 0.6rem;
+    line-height: 1rem;
+    font-weight: 600;
+}
+
+.text-small{
+    font-size: 1rem;
+    line-height: 1.5rem;
+    font-weight: 300;
+}
+
+.text-color-grey{
+    color: grey;
+}
+
+.text-center{
+    text-align: center;
+}
```

Comment:
You could add an empty line at the end of the file to avoid such warnings from Github.